dev-libs/inih/inih-r56.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Simple .INI file parser in C, good for embedded systems"
+DESCRIPTION="a simple .INI file parser written in C. It's only a couple of pages of code, and it was designed to be small and simple"
+HOMEPAGE="https://github.com/benhoyt/inih"
+COPYRIGHT="2009 Ben Hoyt"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/benhoyt/inih/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="4f2ba6bd122d30281a8c7a4d5723b7af90b56aa828c0e88256d7fceda03a491a"
+SOURCE_FILENAME="inih-$portVersion.tar.gz"
+SOURCE_DIR="inih-$portVersion"
+
+ARCHITECTURES="all"
```

Comment:
This doesn't build on primary 32bit, so SECONDARY_ARCHITECTURES="x86" should be added and needed changes below (as $secondaryArchSuffix).